content/pt-br/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+O CRI é uma interface de plug-in que permite ao kubelet usar uma ampla variedade
+ de tempos de execução de contêiner, sem a necessidade de recompilar 
+ os componentes do cluster.
+
+Você precisa de um
+{{<glossary_tooltip text="agente de execução de contêiner" term_id="container-runtime">}} em cada nó em seu cluster, para que o
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} possa iniciar
+{{< glossary_tooltip text="Pods" term_id="pod" >}} e seus contêineres.
+
+{{< glossary_definition prepend="A Interface do agente de execução de contêiner (CRI) é" term_id="container-runtime-interface" length="all" >}}
+
+<!-- body -->
+
+## A API {#api}
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+
+O kubelet atua como um cliente ao se conectar ao agente de execução de contêiner via gRPC. Os endpoints de serviço de imagem e tempo de execução devem estar disponíveis no tempo de execução do contêiner, que pode ser configurado separadamente no kubelet usando os sinais de opções de linha de comando 
```

Comment:
Some bits in this paragraph:
* There are still two occurrences of "tempo de execução" instead of "agente de execução".
* An extra "sinais" ("opções" is a good translation for flags, no need to add "sinais" or "sinalizadores").